Chummer/Classes/ColorManager.cs

```diff
@@ -316,6 +383,27 @@ private static Color InverseGetDarkModeVersion(Color objColor)
             return FromHsla(fltHue, fltNewSaturationHsl, fltNewLightness, objColor.A);
         }
 
+        private static Color GetBrightenedVersion(Color objColor)
+        {
+            // Built-in functions are in HSV/HSB, so we need to convert to HSL to invert lightness.
+            float fltHue = objColor.GetHue() / 360.0f;
+            float fltBrightness = objColor.GetBrightness();
+            float fltLightness = objColor.GetSaturation();
+            fltLightness = Math.Min(fltLightness + 0.2f, 1);
```

Comment:
You sure you want to add 20 to the Saturation of the color instead of multiplying it by some value?

Reply:
I'll change this to use a factor (1.15f). Result looks quite similar, but maybe I don't fully catch the corner cases you have in mind.